cmd/go-swagger/restapi/configure_swagger_bench.go

```diff
@@ -0,0 +1,137 @@
+			return middleware.NotImplemented("operation path_traversal.PathTraversalFront has not yet been implemented")
+		})
+	}
+	if api.SwaggerServerRootHandler == nil {
+		api.SwaggerServerRootHandler = swagger_server.RootHandlerFunc(func(params swagger_server.RootParams) middleware.Responder {
+			return middleware.NotImplemented("operation swagger_server.Root has not yet been implemented")
+		})
+	}
+	if api.SQLInjectionSQLInjectionFrontHandler == nil {
+		api.SQLInjectionSQLInjectionFrontHandler = sql_injection.SQLInjectionFrontHandlerFunc(func(params sql_injection.SQLInjectionFrontParams) middleware.Responder {
+			return middleware.NotImplemented("operation sql_injection.SQLInjectionFront has not yet been implemented")
+		})
+	}
+	if api.SsrfSsrfFrontHandler == nil {
+		api.SsrfSsrfFrontHandler = ssrf.SsrfFrontHandlerFunc(func(params ssrf.SsrfFrontParams) middleware.Responder {
+			return middleware.NotImplemented("operation ssrf.SsrfFront has not yet been implemented")
+		})
+	}
+	if api.UnvalidatedRedirectUnvalidatedRedirectFrontHandler == nil {
+		api.UnvalidatedRedirectUnvalidatedRedirectFrontHandler = unvalidated_redirect.UnvalidatedRedirectFrontHandlerFunc(func(params unvalidated_redirect.UnvalidatedRedirectFrontParams) middleware.Responder {
+			return middleware.NotImplemented("operation unvalidated_redirect.UnvalidatedRedirectFront has not yet been implemented")
+		})
+	}
+	if api.XSSXSSFrontHandler == nil {
+		api.XSSXSSFrontHandler = xss.XSSFrontHandlerFunc(func(params xss.XSSFrontParams) middleware.Responder {
+			return middleware.NotImplemented("operation xss.XSSFront has not yet been implemented")
+		})
+	}
+
+	api.PreServerShutdown = func() {}
+
+	api.ServerShutdown = func() {}
+
+	return setupGlobalMiddleware(api.Serve(setupMiddlewares))
+}
+
+// The TLS configuration before HTTPS server starts.
+func configureTLS(tlsConfig *tls.Config) {
+	// Make all necessary changes to the TLS configuration here.
+}
+
+// As soon as server is initialized but not run yet, this function will be called.
+// If you need to modify a config, store server instance to stop it individually later, this is the place.
+// This function can be called multiple times, depending on the number of serving schemes.
+// scheme value will be set accordingly: "http", "https" or "unix".
+func configureServer(s *http.Server, scheme, addr string) {
+}
+
+// The middleware configuration is for the handler executors. These do not apply to the swagger.json document.
+// The middleware executes after routing but before authentication, binding and validation.
+func setupMiddlewares(handler http.Handler) http.Handler {
+	return handler
+}
+
+func uiMiddleware(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if strings.Index(r.URL.Path, "/assets/") == 0 {
```

Comment:
You should be able to use `if strings.HasPrefix(r.URL.Path, "/assets/") { /*....*/ }` here.

Reply:
I also think it would be nice to save the handler as `var assetHandler` so the code here would be a bit simpler as `assetHandler.ServeHTTP(w,r)`. This is also a nit you can ignore.